modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/PicklistObjectFieldBusinessType.java

```diff
@@ -139,11 +140,23 @@ public Map<String, Object> getProperties(
 				LocalizedValue localizedValue = new LocalizedValue(
 					objectFieldRenderingContext.getLocale());
 
-				localizedValue.addString(
-					objectFieldRenderingContext.getLocale(),
-					String.valueOf(
-						ObjectFieldSettingUtil.getDefaultValue(
-							null, objectField, null)));
+				String defaultValue = String.valueOf(
+					ObjectFieldSettingUtil.getDefaultValue(
+						null, objectField, null));
+
+				if (objectField.isLocalized()) {
+					localizedValue.addString(
+						objectFieldRenderingContext.getLocale(),
+						_jsonFactory.createJSONObject(
+							HashMapBuilder.put(
+								"en_US", defaultValue
+							).build()
+						).toJSONString());
```

Comment:
We should consider the `locale` instead of always setting it to "en_US". Do you agree?

Reply:
Done!